src/utils/urlUtilities.ts

```diff
@@ -23,9 +27,28 @@ export function getFragmentParameter(uri: string, name: string): string {
   }
 }
```

Comment:
It's better to not export the getFragmentParameter function since now we have the more functional getFragments function for that purpose

Reply:
I've updated the codebase using the new getFragments function